docs/user_manual/download_data.md

```diff
@@ -0,0 +1,39 @@
+# Download and prepare the SWIFT data
+
+## Download the data
+
+- Go to NASA’s HEASARC Archive: <https://heasarc.gsfc.nasa.gov/cgi-bin/W3Browse/swift.pl>
+- Next to “UVOT Log”, click on “parameter search form” to open the SWIFT UVOT Data Query form.
+- Use the following parameters (and leave the other parameters to the default values):
+
+    |<!-- -->  | <!-- -->                              |
+    | ------------  | -------------                    |
+    | filter        | `UVW2` OR `UVM2` OR `UVW1`       |
+    | pointing_mode | POINTING                         |
+    | Object Name   | Name of the galaxy, e.g. NGC0628 |
+
+- Click on the “Start Search” button and wait for the query results to load.
+- Check the box “Select All” to select all data.
+- At the bottom: Select “Swift Auxiliary Data (aux)” and “Swift UVOT Data (uvot data)”.
+- Click on the “Create Download Script” button and wait for the new tab to load.
+- Click on the “Download Commands To File” button and wait until the script is downloaded.
+- Create a new directory somewhere on your computer with the name of the galaxy, e.g. “NGC0628”, and create a new directory within that directory with the name “Raw_data”.
+- Move the download script to the “Raw_data” directory and run it:
+
+    ```sh
+    bash browse_download_script.txt
+    ```
+
+Downloading the data can take a while!
+
+- Delete the download script when it has finished running.
```

Comment:
Maybe this should be more optional: "You can delete the download script after it has finished running". Because now it sounds like you have to do that, while it might actually be useful to keep the script. In fact, maybe just remove that line alltogether 😄